test/Conda/EnvironmentYmlSpec.hs

```diff
@@ -6,27 +8,16 @@ import Data.ByteString qualified as BS
 import Data.Map.Strict qualified as Map
 import Data.Yaml (decodeEither', prettyPrintParseException)
 import DepTypes (
-  DepType (CondaType),
+  DepType (CondaType, PipType),
   Dependency (..),
   VerConstraint (CEq),
  )
 import Effect.Grapher
-import GraphUtil (expectDeps)
 import Graphing (Graphing)
-import Strategy.Conda.EnvironmentYml (EnvironmentYmlFile (..), buildGraph)
+import Strategy.Conda.EnvironmentYml (buildGraph)
 import Test.Hspec
 import Test.Hspec qualified as T
-
-dependencyOne :: Dependency
-dependencyOne =
-  Dependency
-    { dependencyType = CondaType
-    , dependencyName = "name"
-    , dependencyVersion = Just (CEq "version1")
-    , dependencyLocations = []
-    , dependencyEnvironments = mempty
-    , dependencyTags = Map.empty
-    }
+import Text.RawString.QQ (r)
 
 dependencyTwo :: Dependency
```

Comment:
I don't think `dependencyTwo` is used outside of `epectedGraph`. Maybe just move it into `expectedGraph`?

Reply:
You're right, I've inlined it.